core/src/test/java/google/registry/tools/GetTldCommandTest.java

```diff
@@ -45,7 +45,7 @@ void testSuccess() throws Exception {
     PremiumList premiumList = persistPremiumList("test", USD, "silver,USD 50", "gold,USD 80");
     persistResource(
         tld.asBuilder()
-            .setDnsAPlusAaaaTtl(Duration.millis(900))
```

Comment:
😬

Well this definitely fixes a pre-existing bug then, huh, where 900 in the YAML looked like it should've been 15 minutes (which would make sense) but it actually represented 900ms.